airbyte-webapp/STYLE-GUIDE.md

```diff
@@ -0,0 +1,56 @@
+# Frontend Style Guide
```

Comment:
Small note, usually I think most repositories call the file `STYLEGUIDE.md` (without the hyphen).